### PR TITLE
Add structured encrypt example (GEA-11385)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: examples/vault/encrypt_structured/pom.xml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        <enabled>false</enabled>
+      </releases>
+    </repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>cloud.pangea</groupId>
+      <artifactId>pangea-sdk</artifactId>
+      <version>3.5.0</version>
```
Comment (pangea-andrest) [inline examples/vault/encrypt_structured/pom.xml:38]: We should update this version before release it

File: examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        var client = new VaultClient.Builder(cfg).build();
+
+        try {
+            // First create an encryption key, either from the Pangea Console or
+            // programmatically as below.
+            var generateResp = client.symmetricGenerate(
+                    new SymmetricGenerateRequest.Builder(
+                            SymmetricAlgorithm.AES256_CFB,
+                            KeyPurpose.ENCRYPTION,
+                            "any unique name").build());
```
Comment (pangea-andrest) [inline examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java:30]: Comment https://github.com/pangeacyber/pangea-go/pull/158#discussion_r1449234519

File: examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            data.put("foo", "bar");
+            data.put("some", "thing");
+
+            // Encrypt
+            var encryptResponse = client.encryptStructured(
+                    new EncryptStructuredRequest.Builder<String, Object, Map<String, Object>>(
+                            encryptionKeyId,
+                            data,
+                            ".foo").build());
+            System.out.println("Encrypted result: " + encryptResponse.getResult().getStructuredData());
```
Comment (pangea-andrest) [inline examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java:44]: Is this going to print a string or just a pointer or something like that?

File: examples/vault/encrypt_structured/pom.xml (hunk 1 above)
Comment (kenany) [reply examples/vault/encrypt_structured/pom.xml:38]: Alright: https://github.com/pangeacyber/pangea-java/pull/188

File: examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java (hunk 3 above)
Comment (kenany) [reply examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java:44]: Actually prints quite nicely:
```
Encrypted result: {foo=f/ZYmVnNXiIxFaKgGVVMPz6STg==, some=thing}
```

File: examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java (hunk 2 above)
Comment (kenany) [reply examples/vault/encrypt_structured/src/main/java/cloud/pangeacyber/examples/App.java]: Fixed.

**Review by pangea-andrest — DISMISSED** [review]

File: examples/vault/encrypt_structured/pom.xml (hunk 1 above)
Comment (kenany) [reply examples/vault/encrypt_structured/pom.xml]: Done.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**